docs/reference/ilm/index.asciidoc

```diff
@@ -46,6 +46,16 @@ to a single shard.
 . After 7 days, move the index into the cold stage and move it to less expensive
 hardware.
 . Delete the index once the required 30 day retention period is reached.
+
+[IMPORTANT]
+===========================
+{ilm} does not support mixed-cluster usage. Usage of new policies using
+new actions that do not exist in the oldest version of a cluster with
+mixed-versioned nodes. Although it may be possible to create such new
```

Comment:
This sentence doesn't have a verb - this should say something more like (new text in brackets): 
```
Usage of new policies using
new actions that do not exist in the oldest version of a cluster with
mixed-versioned nodes [will cause errors if those actions are run on nodes which do not support those new actions].
```

Reply:
To be clear, that may not be what we want to say, but as written the sentence is confusing.

Reply:
haha woops. edited super last minute and didn't fully read it